README.md

```diff
@@ -60,9 +60,9 @@ LoggingSystem.bootstrap(StreamLogHandler.standardError)
 For further information, please check the [API documentation][api-docs].
 
 <a name="backends"></a>
-#### Selecting a logging backend implementation (applications only)
```

Comment:
the (applications only) part is important so we should find a place to say that

Reply:
Updated